pkg/oss/controllerserver.go

```diff
@@ -61,6 +61,10 @@ func getOssVolumeOptions(req *csi.CreateVolumeRequest) *Options {
 			ossVolArgs.UseSharedPath = true
 		} else if key == "authtype" {
 			ossVolArgs.AuthType = value
+		} else if key == "rolename" {
+			ossVolArgs.RoleName = value
```

Comment:
The user is responsible for creating this RAM role and authorizing the desired OSS permission. right?